backend/lcfs/web/api/user/services.py

```diff
@@ -125,16 +125,16 @@ async def get_all_users(self, pagination: PaginationRequestSchema) -> UsersSchem
         Get all users
         """
         users, total_count = await self.repo.get_users_paginated(pagination=pagination)
-        if len(users) == 0:
-            raise DataNotFoundException("No users found")
         return UsersSchema(
             pagination=PaginationResponseSchema(
                 total=total_count,
                 page=pagination.page,
                 size=pagination.size,
-                total_pages=math.ceil(total_count / pagination.size),
+                total_pages=(
+                    math.ceil(total_count / pagination.size) if total_count > 0 else 0
+                ),
             ),
-            users=users,
+            users=users if users else [],
```

Comment:
Not need to add contidition, `get_users_paginated` is returning  [] if no users or array with UserProfile entities